murano.yaml

```diff
@@ -26,6 +27,14 @@ info:
 #
 ####
 
+options:
+  # Following options defines the update strategy when applying the template to a solution
+  # This only applies to: Webservice/Websocket endpoints, Asset files & Modules
+  # Service configuration & Eventhandler always follow a merge behavior
+  # merge: true, # To keep all existing items from solution that are not present in this template
```

Comment:
I think we should uncomment this line in order to keep all existing files?

Reply:
```suggestion
  merge: true, # To keep all existing items from solution that are not present in this template
```